src/aac/genplug.py

```diff
@@ -117,6 +118,10 @@ def set_filename_value(template: TemplateOutputFile):
             template.template_name, plugin_implementation_name
         )
 
+    def set_parent_directory_value(template: TemplateOutputFile):
+        if template.template_name in template_parent_directories:
+            template.parent_dir = template_parent_directories[template.template_name]
```

Comment:
This can be one-lined like such:
```suggestion
template.parent_dir = template_parent_directories.get(template.template_name) or template.parent_dir
```

Reply:
Resolved with ab43e25